core/text.py

```diff
@@ -43,6 +43,14 @@ def format_item(self, item, ql=None, with_icon=True):
 
         return result
 
+    def generate_item(self, item, ql, synonym=None):
```

Comment:
It appears the `synonym` field is doing nothing, as the output is the exact same either way, so it should be removed.

Reply:
also, is this a method that is going to be used by other modules? right now it's just used by the alien module in one place, and it seems pretty specific in that it hard codes a ql of 100, so it should probably be moved into the alien module.

Reply:
The "synonym" field is being used to identify specific icons/itemrefs in fully translatable blobs (currently, only the Alien-Module has one) indeed, the ql 100 was some leftover... 
i'm currently using it in some custom modules (i usually have most changes in my own production for an few weeks, before commiting - but somehow nobody discovered the QL thingie...) 

i decided to move it into text.py, because its kinda likely, that it will be needed in more modules, later on - the list is still long.